### ISSUE TITLE
Firefox SyntaxError: missing = in const declaration

### ISSUE THREAD

**Issue by isurindu** [ecrmnn/collect.js#105]
i'm getting this error on firefox 

SyntaxError: missing = in const declaration

module.exports = function except(properties) {
  const collection = {};

  for (const prop in this.items) {
    if (properties.indexOf(prop) === -1) {
      collection[prop] = this.items[prop];
    }
  }

  return new this.constructor(collection);
};

**Comment by ajaymathur** [comment]
reference : [MDN link](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Errors/Missing_initializer_in_const)

**Comment by g-plane** [comment]
@isurindu How do you import this library?

**Comment by isurindu** [comment]
like this 

window.collect = require('collect.js')

**Comment by g-plane** [comment]
Well, browsers don't support CommonJS, so you can't use `require` directly. You can use some tools like Webpack to bundle it. To be delightful, this library may support UMD later, which means you can use `collect` as a global variable.

**Comment by isurindu** [comment]
i'm using laravel mix

**Comment by g-plane** [comment]
Oh, sorry for my misunderstanding.

**Comment by ecrmnn** [comment]
I'm not able to reproduce this. What am I missing?

```js
//file:  resources/assets/js/app.js

window.collect = require('collect.js');
```

```js
//file: webpack.mix.js

let mix = require('laravel-mix');

/*
 |--------------------------------------------------------------------------
 | Mix Asset Management
 |--------------------------------------------------------------------------
 |
 | Mix provides a clean, fluent API for defining some Webpack build steps
 | for your Laravel application. By default, we are compiling the Sass
 | file for the application as well as bundling up all the JS files.
 |
 */

mix.js('resources/assets/js/app.js', 'public/js')
   .sass('resources/assets/sass/app.scss', 'public/css');
```

```html
<!doctype html>
<html lang="{{ app()->getLocale() }}">
    <head>
        <meta charset="utf-8">
        <meta http-equiv="X-UA-Compatible" content="IE=edge">
        <meta name="viewport" content="width=device-width, initial-scale=1">

        <title>Laravel</title>

        <!-- Fonts -->
        <link href="https://fonts.googleapis.com/css?family=Raleway:100,600" rel="stylesheet" type="text/css">
    </head>
    <body>
        <div class="flex-center position-ref full-height">
            @if (Route::has('login'))
                <div class="top-right links">
                    @if (Auth::check())
                        <a href="{{ url('/home') }}">Home</a>
                    @else
                        <a href="{{ url('/login') }}">Login</a>
                        <a href="{{ url('/register') }}">Register</a>
                    @endif
                </div>
            @endif

            <div class="content">
                <div class="title m-b-md">
                    Laravel
                </div>

                <div class="links">
                    <a href="https://laravel.com/docs">Documentation</a>
                    <a href="https://laracasts.com">Laracasts</a>
                    <a href="https://laravel-news.com">News</a>
                    <a href="https://forge.laravel.com">Forge</a>
                    <a href="https://github.com/laravel/laravel">GitHub</a>
                </div>
            </div>
        </div>
        <script src="{{ mix('js/app.js') }}"></script>
    </body>
</html>
```

```bash
npm run hot
```

**Comment by ecrmnn** [comment]
@isurindu 

I've just published version 3.0.31.

Could you try to reinstall your node modules? And try again?

```bash
rm -rf node_modules/
npm install
```
